ansible/group_vars/all.yml

```diff
@@ -69,10 +69,9 @@ virtual_machines:
   rhcos:
     project: rhcos-cloud
     images:
-      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.17') }}"
-      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.16') }}"
+      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.19') }}"
+      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.18') }}"
```

Comment:
Should we think of a way to automate bumping these versions?

Reply:
Yeah that would be nice. In the meantime I'm going to add it as a step in the internal wiki process `Add support for new OpenShift version`.